collectors/ebpf.plugin/README.md

```diff
@@ -39,787 +35,826 @@ The following two features from the Linux kernel are used by Netdata to run eBPF
 ```
 
-#### MD flush
-
-To collect data related to Linux multi-device (MD) flushing, the following kprobe is used:
-
--  `kprobe/md_flush_request`: called whenever a request for flushing multi-device data is made.
-
-### Disk
-
-The eBPF plugin also shows a chart in the Disk section when the `disk` thread is enabled.
+### Sync configuration
 
-#### Disk Latency
-This will create the chart `disk_latency_io` for each disk on the host. The following tracepoints are used:
+The sync configuration has specific options to disable monitoring for syscalls, as default option all syscalls are
+monitored.
 
-- [`block/block_rq_issue`](https://www.kernel.org/doc/html/latest/core-api/tracepoint.html#c.trace_block_rq_issue):
-  IO request operation to a device drive.
-- [`block/block_rq_complete`](https://www.kernel.org/doc/html/latest/core-api/tracepoint.html#c.trace_block_rq_complete):
-  IO operation completed by device.
+```conf
+[syscalls]
+    sync = yes
+    msync = yes
+    fsync = yes
+    fdatasync = yes
+    syncfs = yes
+    sync_file_range = yes
+```
 
-Disk Latency is the single most important metric to focus on when it comes to storage performance, under most circumstances.
-For hard drives, an average latency somewhere between 10 to 20 ms can be considered acceptable. For SSD (Solid State Drives),
-in most cases, workloads experience less than 1 ms latency numbers, but workloads should never reach higher than 3 ms.
-The dimensions refer to time intervals.
+### Filesystem configuration
 
-### Filesystem
+The filesystem configuration has specific options to disable monitoring for filesystems, by default all filesystems are
+monitored.
 
-This group has charts demonstrating how applications interact with the Linux kernel to open and close file descriptors.
-It also brings latency charts for several different filesystems.
+```conf
+[filesystem]
+    btrfsdist = yes
+    ext4dist = yes
+    nfsdist = yes
+    xfsdist = yes
+    zfsdist = yes
+```
 
-#### Latency Algorithm
-We calculate the difference between the calling and return times, spanning disk I/O, file system operations (lock, I/O),
-run queue latency and all events related to the monitored action.
+The ebpf program `nfsdist` monitors only `nfs` mount points.
 
-#### ext4
+## Troubleshooting
 
-To measure the latency of executing some actions in an
-[ext4](https://elixir.bootlin.com/linux/latest/source/fs/ext4) filesystem, the
-collector needs to attach `kprobes` and `kretprobes` for each of the following
-functions:
+If the eBPF collector does not work, you can troubleshoot it by running the `ebpf.plugin` command and investigating its
+output.
 
-- `ext4_file_read_iter`: Function used to measure read latency.
-- `ext4_file_write_iter`: Function used to measure write latency.
-- `ext4_file_open`: Function used to measure open latency.
-- `ext4_sync_file`: Function used to measure sync latency.
+```bash
+cd /usr/libexec/netdata/plugins.d/
+sudo su -s /bin/bash netdata
+./ebpf.plugin
```

Comment:
`ebpf.plugin` also needs special permission to run, because it is monitoring kernel metrics, please, add `sudo su -s` for it too.

Why are you adding `netdata` here ?

Reply:
Hi @thiagoftsm , 
I think this section got marked as "green - added" because I did cut and paste a lot of things. 

I didn't touch this code example and I'm not exactly sure why it looks like this. 

Feel free to tell me what you would expect to see here, so I can learn from you. :)